r7-cert.yaml

```diff
@@ -2,20 +2,20 @@ ceremony-type: intermediate
 pkcs11:
   module: /usr/lib/x86_64-linux-gnu/softhsm/libsofthsm2.so
   pin: 1234
-  signing-key-slot: 1094195990
+  signing-key-slot: 1307844626
   signing-key-label: root-x1
 inputs:
   issuer-certificate-path: root-x1.cert.pem
-  public-key-path: int-r3.key.pem
+  public-key-path: int-r7.key.pem
 outputs:
-  certificate-path: int-r3.cert.pem
+  certificate-path: int-r7.cert.pem
 certificate-profile:
   signature-algorithm: SHA256WithRSA
-  common-name: Example RSA 1
-  organization: Example
+  common-name: (FAKE) R7
```

Comment:
Why `R7` and `R8`? We don't have an `R5` or `R6`.

Reply:
Because the ECDSA certs are using the numbers 5 and 6. I don't love it either, but I remember it being pitched this way a long time ago. Happy to revisit that discussion and do `E3`, `E4`, `R5`, `R6` instead, though.

Reply:
I believe I pitched that. Here's my reasoning: single-letter (or single-number) differences are easy to typo or misremember - typing E5 when you meant R5, for instance. We should therefore choose letter-number combinations so they are not maximally dense. In other words, we should choose them such that there are intentionally invalid combinations. If someone types "my site is serving the E3 certificate," it's easy to spot that there must be a typo or confusion. If E3 actually existed, there might be a prolonged back-and-forth where someone misunderstands that as having actually meant E3.